projects/igniteui-angular/src/lib/services/overlay/position/auto-elastic-position-strategy.ts

```diff
@@ -0,0 +1,28 @@
+import { ConnectedFit, Size } from './../utilities';
+import { AutoPositionStrategy } from './auto-position-strategy';
+
+/**
+ * TODO
+ */
+export class AutoElasticPositionStrategy extends AutoPositionStrategy {
+    /** @inheritdoc */
+    position(contentElement: HTMLElement, size: Size, document?: Document, initialCall?: boolean): void {
+        this._forceFit = true;
```

Comment:
we do not need this, or I cannot imagine a case when this would help :)

Reply:
Now using method shouldFitInViewPort(...)